### ISSUE TITLE
Choose networks

### ISSUE THREAD

**Issue by Wingee** [bigpaulie/yii2-social-share#1]
Would it be possible to select which networks we want to share to? For example I don't want to share to LinkedIn

Thanks

**Comment by fatalkiss** [comment]
Go to widget.php and comment in array $networks this row  
`'linkedin' => 'https://www.linkedin.com/shareArticle?mini=true&url={url}',`

**Comment by fatalkiss** [comment]
Or you can add in Widget.php this row before `public $url;`
 `public $missButton = array();`
Next in function `parseTemplate` add  on top

```
  if (in_array($network,$this->missButton)) {
          return false;
  }
```

And then usage it 

```
  <?php
    echo Share::widget([
            'url' => 'site.com',
            'text' => 'Click to share on {network}',
            'type' => 'small',
            'tag' => 'ul',
            'template' => '<li>{button}</li>',
            'missButton'=>['linkedin','google-plus'],
            'htmlOptions' => [
            'id' => 'social_button',
        ],
        ]);
     ?>
```

Good luck

**Comment by bigpaulie** [comment]
@fatalkiss , if you submit a pull request I will merge it with the repository, I'm very busy at work lately and I don't have much time left these days ...

**Comment by bigpaulie** [comment]
Solved by #2
